lutris.spec

```diff
@@ -51,12 +51,14 @@ BuildRequires: fdupes
 %ifarch x86_64
 Requires:       mesa-vulkan-drivers(x86-32)
 Requires:       vulkan-loader(x86-32)
+Requires:       mesa-libGL(x86-32)
```

Comment:
This was removed because there is no such package SUSE. It should be added under `%if 0%{?fedora}`/`%ifarch x86_64` instead (along with `mesa-dri-drivers(x86-32)`).